move/squid/sources/squid/transfers.move

```diff
@@ -1,16 +1,21 @@
 module squid::transfers;
 
-use its::interchain_transfer_ticket::InterchainTransferTicket;
-use its::its;
-use its::token_id::{Self, TokenId};
-use relayer_discovery::transaction::{Self, MoveCall};
-use squid::squid::Squid;
-use squid::swap_info::SwapInfo;
 use std::ascii::{Self, String};
 use std::type_name;
+
 use sui::bcs::{Self, BCS};
 use sui::coin;
+use sui::clock::Clock;
+
+use axelar_gateway::gateway::Gateway;
+
+use relayer_discovery::transaction::{Self, MoveCall};
+
+use its::its::{Self, ITS};
+use its::token_id::{Self, TokenId};
 
+use squid::squid::Squid;
+use squid::swap_info::SwapInfo;
```

Comment:
install the latest version of the [formatter](https://interoplabs.slack.com/archives/C05T6A1LH8T/p1726654231699699?thread_ts=1726213849.107779&cid=C05T6A1LH8T) and run on the files again. It groups imports differently